docs/contribute.rst

```diff
@@ -3,32 +3,205 @@
+    contain enough information to decide if it is valid or not and ask on the
+    ticket for the required information to go forward. We will re-triage all
+    tickets that have the label **Needed: more information** assigned. If the
+    original reporter left new information we can try to re-categorize the
+    ticket. If the reporter did not come back to provide more required
+    information after a long enough time, we will close the ticket (this will be
+    roughly about two weeks).
+
+    Examples:
+
+    - *My builds stopped working. Please help!*
+      Ask for a link to the build log and for which project is affected.
+
+… is a valid enhancement proposal
+    If the ticket contains an enhancement proposal that aligns with the goals
+    of Read the Docs, then add the label **Enhancement**. If the proposal
+    seems valid but requires further discussion between core contributors
+    because there might be different possibilities on how to implement the
+    enhancement, then also add the label **Needed: design decision**.
+
+    Examples:
+
+    - *Improve documentation about MKdocs integration*
+    - *Provide better integration with service XYZ*
+    - *Refactor module X for better readability*
+    - *Achieve world domination* (also needs the label **Needed: design
+      decision**)
+
+… is a valid problem within the code base:
+    If it's a valid bug, then add the label **Bug**. Try to reference related
+    issues if you come across any.
+
+    Examples:
+
+    - *Builds fail if conf.py contains non-ascii letters*
+
+… is a currently valid problem with the infrastructure:
+    Users might report about web server downtimes or that builds are not
+    triggered. If the ticket needs investigation on the servers, then add the
+    label **Operations**.
+
+    Examples:
+
+    - *Builds are not starting*
+
+… is a question and needs answering:
+    If the ticket contains a question about the Read the Docs platform or the
+    code, then add the label **Support**.
+
+    Examples:
+
+    - *My account was set inactive. Why?*
+    - *How to use C modules with Sphinx autodoc?*
+    - *Why are my builds failing?*
+
+… requires a one-time action on the server:
+    Tasks that require a one time action on the server should be assigned the
+    two labels **Support** and **Operations**.
+
+    Examples:
+
+    - *Please change my username*
+    - *Please set me as owner of this abondand project*
+
+After we finished the initial triaging of new tickets, no ticket should be left
+without a label.
+
+Additional labels for categorization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Additionally to the labels already involved in the section above, we have a
+few more at hand to further categorize issues.
+
+*Priority: High*
+    If the issue is urgent, assign this label. In the best case also go forward to
+    resolve the ticket yourself as soon as possible.
+
+*Community Effort*
```

Comment:
I'm not sure I really understand this label. My assumption is that all none assigned issues could be taken on by the community? Maybe the `accepted` label is the label that community members should look at for determining whether an issue is valid for working on?